tests/integration/test_interaction.py

```diff
@@ -562,6 +582,81 @@ def test_submit_button_shown(self):
 
         self.assertEqual(submit_button.get_attribute('disabled'), None)
 
+    def test_misplaced_items_returned_to_bank(self):
```

Comment:
@e-kolpakov I would not block the PR for this, it is more a preference than anything.